image_tools_py/cam2image_py.py

```diff
@@ -0,0 +1,200 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# System imports
+import argparse
+import sys
+
+# Local imports
+import burger_py
+
+# OpenCV imports
+import cv2
+
+# ROS2 imports
+import rclpy
+from rclpy.qos import qos_profile_system_default, qos_profile_sensor_data
+import sensor_msgs.msg
+import std_msgs.msg
+
+
+# Convert an OpenCV matrix encoding type to a string format recognized by
+# sensor_msgs::Image
```

Comment:
@clalancette nit: could be a docstring for the function below.

Reply:
Sure, makes sense.